src/org/zaproxy/zap/extension/api/CoreAPI.java

```diff
@@ -198,10 +199,12 @@ public CoreAPI() {
 				OTHER_SEND_HAR_REQUEST,
 				new String[] { PARAM_REQUEST },
 				new String[] { PARAM_FOLLOW_REDIRECTS }));
+		this.addApiOthers(new ApiOther(OTHER_SCRIPT_JS, false));
```

Comment:
Do we really want this exposed in the "normal" endpoints (which end up in the client APIs)? Why not just as API shortcut?